src/input-range-slider/input-range-slider.tsx

```diff
@@ -132,8 +136,24 @@ export const InputRangeSlider = ({
         );
     };
 
+    const formatIndicationLabel = () => {
+        return (
+            <LabelText>
+                Searching within&nbsp;
+                {indicationLabelPrefix}
+                {selection}
+                {indicationLabelSuffix}
+            </LabelText>
```

Comment:
this label should be generic so the text `Searching within` should not be hardcoded
```suggestion
            <LabelText>
                {indicationLabelPrefix}
                {selection}
                {indicationLabelSuffix}
            </LabelText>
```

Reply:
Noted haha, i always thought that prefix indicated value infront of changing value, etc currency, measurement etc. Thanks for remind